src/openapi-v31-fundamentals/module-5-extending-openapi.md

```diff
@@ -0,0 +1,304 @@
+
+Merging two examples from the specification:
+
+```yaml
+overlay: 1.0.0
+info:
+  title: Targeted Overlays
+  version: 1.0.0
+actions:
+  - target: $.paths['/foo'].get
+    update:
+      description: This is the new description
+  - target: $.paths['/bar'].get
+    update:
+      description: This is the updated description
+  - target: $.paths['/bar']
+    update:
+      post:
+        description: This is an updated description of a child object
+        x-safe: false
+  - target: $.paths.*.*.parameters[?@.name == 'sandbox-token']
+    remove: true
+```
+
+The code snippet indicates the OpenAPI document should be updated with new values, namely:
+
+- The `description` property at `$.paths['/foo'].get` is updated with a new value.
+- The `description` property at `$.paths['/bar'].get` is updated with a new value.
+- The Path Item at `$.paths['/bar']` is updated with a new `post` Operation object.
+- A parameter that matches the name `sandbox-token` in any Path Item and any Operation is removed.
+
+This clearly demonstrates a deterministic approach to merging information, and serves the needs of the API lifecycle view of creating and using OpenAPI that we've discussed in modules 3 and 4. Organizations can leverage this specification to update OpenAPI document at different stages in the lifecycle, and always expect consistent results when repeating the lifecycle stages. Overlays therefore serves to take the guesswork out of how to approach this, which should be of considerable benefit to tooling makers and consumers of OpenAPI documents.
+
+> Overlays is currently an implementers draft created to provide a means for the community to test the specification as it stands. This course will be updated in the future should any aspect change drastically.
+
+### Workflows
+
+In the examples we've looked at throughout this course we've always been looking at a single API described by an OpenAPI document. What about cases where there are multiple APIs and those APIs are linked together to form a use cases or function that requires orchestration? It is for such use cases that the [Workflows SIG](https://github.com/OAI/sig-workflows) was formed. The goal of this SIG is:
+
+_"...propose an enhancement to the current OpenAPI, or accompanying specification (or means), that can define sequences of calls and their dependencies to be expressed in the context of delivering a particular outcome or set of outcomes."_
+
+This goal has manifested itself in version 1.0.0 of the [Workflows Specification](https://github.com/OAI/sig-workflows/blob/main/versions/1.0.0.md). API providers can use this specification to provide a schematic representation of how to call several APIs and link them together using the data returned. Each Workflow document references one-or-more source documents, which at version 1.0.0 can either be an OpenAPI document or a Workflow document.
```

Comment:
Just an FYI - There has been lots of discussion recently in the OAI regarding what do we name something created conforming to OpenAPI and/or Workflows.......and for most use cases _description_ over _document_ seems to be the choice. Not sure if this is something you want to align to as part of the course material.